api/ValidityState.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#the-constraint-validation-api:validitystate-3",
         "support": {
           "chrome": {
-            "version_added": "15"
+            "version_added": "3"
```

Comment:
This matches the many `validity` properties on `HTMLButtonElement` and elsewhere.